code/modules/mining/lavaland/cain_and_abel/_cain_and_abel.dm

```diff
@@ -181,6 +201,17 @@
 	animate(wisp_to_remove, alpha = 0, time = 0.2 SECONDS)
 	QDEL_IN(wisp_to_remove, 0.2 SECONDS)
 
+/obj/item/cain_and_abel/proc/check_wield(mob/living/user)
+	if (!istype(user))
+		return TRUE
+	var/active_item = (src == user.get_active_held_item())
+	if (active_item ? user.get_inactive_held_item() : user.get_active_held_item())
```

Comment:
Evil ternary in if statement

Reply:
Better than doing a split if its an active item early, or pulling it out into a variable imo